### PR TITLE
Dockerise national rail

### LINKED ISSUE #72: Dockerise NationalRail
### Task
- [x] Dockerfile for national rail pipeline

### User Story
As a developer, I want to create docker image of the national rail pipeline so that it can be containerised and pushed to the cloud.

### PR DESCRIPTION
**Pull request by Zhi-704**
Close #72 by adding a dockerfile, the pipeline has now been pushed onto AWS and deployed as a lambda function. The function is then triggered every 5 minutes.

### CODE REVIEW

File: national_rail/national_rail.py [hunk 1]
```diff
@@ -1,23 +1,45 @@
...
 
-FILENAME = "data.xml"
+FILENAME = "/tmp/data.xml"
 
-if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO,
-                        format="%(asctime)s - %(levelname)s - %(message)s")
 
-    get_national_rail_data(FILENAME)
+def main(event, context):  # pylint: disable=unused-argument
```
Comment (A-Faris) [inline national_rail/national_rail.py]:
```suggestion
def main(_event, _context):
```

**Review by A-Faris — DISMISSED** [review]
This is amazing

File: national_rail/national_rail.py (hunk 1 above)
Comment (Zhi-704) [reply national_rail/national_rail.py]: Would this still work?

Comment (A-Faris) [reply national_rail/national_rail.py]: Yes

**Comment by Zhi-704** [comment]
Bad bot

**Review by A-Faris — APPROVED** [review]
Yeah it's great

**Pull request merged**